### DISCUSSION TITLE
boundary_id_offset

### DISCUSSION THREAD

**Discussion by tctheodosiou** [idaholab/moose#25292] [Q&A Meshing] [answered]
Hello,
Is it possible to have MOOSE keep track of boundary IDs and create proper numbering (i.e. without conflicts).
Because setting manually the boundary_id_offset for every object created, can be the source of many errors especially if the input file is reorganized (e.g. an object is replaced by another one with different number of boundaries etc.).

Thank you

**Comment by GiudGiud** [comment, marked as answer]
Hello

Not right now for most meshing objects. Many have special accomodations to avoid overriding boundary/block ids (stitching / combiners for examples)
The reactor module RGMB workflow makes sure to handle that for the user. But the generality offered by the other general meshing tools does not allow for that automatic setting currently.  

Guillaume
